### PR TITLE
Add a rudimentary warning to the link share dialog

### PR DESCRIPTION
**Pull request by yuvipanda**
Anyone with the token has full access to your server -
including all your files - until you restart your server.
We must make sure users know this.

Moves the link into a textbox that autoselects on click,
rather than just a link. Users don't actually want to click this -
they only want to copy this. It also provides stylistic relief
given we have so much more text there now.

image

This is how it looks now.

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks!

File: src/index.ts [hunk 1]
```diff
@@ -67,20 +68,54 @@ const plugin: JupyterFrontEndPlugin<void> = {
...
+        // Warn users of the security implications of using this link
+        // FIXME: There *must* be a better way to create HTML
+        const warning = document.createElement('div');
+
+        const warningHeader = document.createElement('h3');
+        warningHeader.innerText = trans.__('Security warning!');
+        warning.appendChild(warningHeader);
+
+        const messages = [
+          'Anyone with this link has full access to your notebook server, including all your files!',
```
Comment (jtpio) [inline src/index.ts:91]: Since these are user facing strings, it would be great if we could use the `ITranslator` to handle localization.

Comment (jtpio) [reply src/index.ts:91]: Never mind, it's done below, thanks!

Comment (yuvipanda) [reply src/index.ts:91]: yeah, i did them all together!

**Comment by yuvipanda** [comment]
Failing check is unrelated again I think.

**Comment by yuvipanda** [comment]
Check has passed! \o/

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**